src/main/java/peoplesoft/ui/MainWindow.java

```diff
@@ -62,65 +69,33 @@ public MainWindow(Stage primaryStage, Logic logic) {
 
         // Configure the UI
         setWindowDefaultSize(logic.getGuiSettings());
-
-        setAccelerators();
-
-        helpWindow = new HelpWindow();
-    }
-
-    public Stage getPrimaryStage() {
-        return primaryStage;
-    }
-
-    private void setAccelerators() {
```

Comment:
If there is a shortcut we need to include it in the userguide, and there really isn't any prevalent mental model of what F1 should do in user's minds. Since they are using command line they shouldnt need shortcuts anymore

Reply:
yeah i guess this is really just a windows thing